electron/legendary/games.ts

```diff
@@ -292,6 +294,26 @@ class LegendaryGame extends Game {
     return await execAsync(command)
   }
 
+  private showDiscordRPC(gameTitle : string) {
+    let os = 'Unknown'
+
+    if (process.platform === 'linux') {
```

Comment:
Even though using if is not a problem, you could use a switch here.

Reply:
Or even a simple Object map.

Reply:
E.g.
{
win32: "Windows"
}